enginetest/queries/script_queries.go

```diff
@@ -6244,3 +6244,463 @@ var DropDatabaseScripts = []ScriptTest{
 		},
 	},
 }
+
+var ZeroTime = time.Date(0000, time.January, 1, 0, 0, 0, 0, time.UTC)
+var SetupTime = time.Date(2000, time.January, 1, 12, 0, 0, 0, time.UTC)
```

Comment:
I would name these after the time they describe, e.g. Jan1Noon etc.

Reply:
Could also make them const

Reply:
Renamed, but I golang does not like `time.Date()` as a const